dpctl/dparray.py

```diff
@@ -0,0 +1,224 @@
+import numpy as np
+from inspect import getmembers, isfunction, isclass, isbuiltin
+from numbers import Number
+from types import FunctionType as ftype, BuiltinFunctionType as bftype
+import sys
+#import importlib
+#import functools
+import inspect
+
+debug = False
+
+def dprint(*args):
+    if debug:
+        print(*args)
+        sys.stdout.flush()
+
+import dpctl
+from dpctl.memory import MemoryUSMShared
```

Comment:
Imports usually are on top of the file. Is it a specific case?

Reply:
Seems to be at the top now.